1904-10-31.xml

```diff
@@ -0,0 +1,2512 @@
+                        season as he is grealty interested in the success of the syndicate.</p>
+                </div>
+                <div type="item">
+                    <head>NEW BANK IN EGYPT.</head>
+                    <p>Mr. Paccelli, manager of the Bance di Roma, along with Com. Tittoni, a cousin
+                        of the Italian Minister for Foreign Affairs, and Mr. Benucci, recently
+                        arrived at Alexandria in order to consider the advisability of establishing
+                        a branch of the bank in egypt. We understand that the opinion formed by
+                        these gentlemen of the prospects of a branch here is favourable to its
+                        establishment, and there is every likelihood of the proposal being carried
+                        through.</p>
+                </div>
+                <div type="item">
+                    <head>IRRIGATION PUMPS.</head>
+                    <p>Gwynne's (Limitied) have obtained an important contract for the supply of
+                        irrigation pumping machinery for the great syndicate,of which Mr. Leigh Hunt
+                        is the moving spirit, for cotton growing in the Sudan. This firm have over
+                        half a century's experience as hydraulic engineers, and have turned out
+                        centrifugal pumpimg machinery capable of discharging 363,636 cubic metres of
+                        363 rivers as large as the Thames, or 72 rivers as large as the Nile.</p>
+                    <p>Mr. A. L. Webb, of the Irrigation Department, when in Holland in 1897, on
+                        behalf of the Egyptian Government, gave and excellent report as to the
+                        working of Gwynne's machinery in that country. In his work on Egyptian
+                        Irrigation Sir William Willcocks remark:– "Indeed if it were not that the
+                        quality of English engines is so high and the Gwynne's pumps themselves
+                        nearly indestructible, pumping machinery would have succumbed to the Persian
+                        wheels." (See page 246 "Egyptian Irrigation, 1889"). Some of these machines
+                        have been at work for over fifty years.</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-who03">
+                    <head>WINDSOR HOTEL</head>
+                    <p>Largest 1st class and most comfortable Hotel in Alexandria. Facing the sea
+                        Central position. Under English managemnt.</p>
+                    <p>81-11-904</p>
+                </div>
+            </div>
+            <pb n="3"/>
+            <div type="page" n="3" facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p3.jpg"> 
+                <cb n="1"/>
+                <div type="section" feature="wire">
```

Comment:
this section is complex to format. Treat cables as individual divs (type="cable"), each with its own <dateline> and <title>. This section is sometimes spread over two pages; when this is the case, you can use the next and prev attributes. Take a peak at a finished file for an example.